packages/maskbook/src/plugins/GoodGhosting/UI/GameAction.tsx

```diff
@@ -45,7 +45,7 @@ export function GameAction(props: GameActionProps) {
         value: tokenDetailed,
         loading: loadingToken,
         error: errorToken,
-    } = useERC20TokenDetailed(DAI[chainId]?.address)
+    } = useERC20TokenDetailed(DAI[chainId].address)
```

Comment:
`DAI[chainId]` could be`undefined`, since the chain id could be any integer value.